docs/reference/files/requirements-txt-file.rst

```diff
@@ -0,0 +1,13 @@
+.. _requirements-txt-file:
+
+
+``requirements.txt`` file
+=========================
+
+The ``requirements.txt`` file is a standard Python `requirements file
+<https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format>`_
+used to declare and pin the version of any Python libraries required by
+a charm in production. This will be pre-populated with
+:literalref:`ops <https://ops.readthedocs.io/en/latest/>`_. Any dependencies
```

Comment:
```suggestion
:literalref:`ops <https://ops.readthedocs.io/en/latest/>`. Any dependencies
```

Ops should also probably be a reusable link in docs/reuse/links.txt tbh

Reply:
This link is incorrect. We should be pointing to the actual reference page in ops, or this shouldn't be a literalref.